### PR TITLE
Feat dark mode

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Implemented Dark Mode
- landing page
- documentation page

### CODE REVIEW

File: components/style.css [hunk 1]
```diff
@@ -229,6 +234,38 @@ i:hover {
...
+    display: none;
+}
+
+body.dark-mode .logo-dark {
+    display: block;
+}
+
+body.dark-mode .hero {
+    
+}
```
Comment (prachi-sahani) [inline components/style.css]: Empty style block can be removed

**Review by prachi-sahani — COMMENTED** [review]
code is well structured and good to merge

File: components/style.css (hunk 1 above)
Comment (hinanshisuthar) [reply components/style.css]: kept it blank as I am still figuring how to have perfect image for black mode. Cool, I'll remove it for now.

**Review by stutisk — APPROVED** [review]
Looks good from my side to merge, great work hinanshi !

**Pull request merged**